bundles.rst

```diff
@@ -182,3 +179,4 @@ Learn more
     bundles/*
 
 .. _`third-party bundles`: https://github.com/search?q=topic%3Asymfony-bundle&type=Repositories
+.. _`SensioGeneratorBundle`: https://github.com/sensiolabs/SensioGeneratorBundle
```

Comment:
What about linking to http://symfony.com/doc/current/bundles/SensioGeneratorBundle/index.html instead?